stable/concourse/templates/secrets.yaml

```diff
@@ -18,49 +18,64 @@ data:
   {{- if not .Values.postgresql.enabled }}
   postgresql-uri: {{ template "concourse.secret.required" dict "key" "postgresqlUri" "isnt" "postgresql.enabled" "root" . }}
```

Comment:
I don't think `CONCOURSE_POSTGRES_DATA_SOURCE` works anymore in 4.1, and it's been removed in https://github.com/helm/charts/pull/7804/files#diff-7198763aa1575d2ac19c7b12a2794978L82, so we need new secrets to store use DB username and password here.